tests/ioctl_kvm_run_auxstr_vcpu.c

```diff
@@ -12,20 +12,35 @@
 
 #if need_print_KVM_RUN
 
+#define need_print_KVM_RUN_MORE 1
```

Comment:
I don't see why `need_print_KVM_RUN_MORE` is being defined in this file.

Reply:
need_print_KVM_RUN_MORE is redundant.I will remove this.